acrobat/blocks/eventwrapper/eventwrapper.js

```diff
@@ -59,12 +54,11 @@ export default function init(element) {
   };
 
   const handleEvents = (e, jobData, converter, verb) => {
-    let parser = bowser.getParser(window.navigator.userAgent);
-    let browserName = parser.getBrowserName();
+    const { name: browserName, isMobile } = getBrowserData();
     let extID;
     if (e === PROCESS_START) converterAnalytics();
-    if (e === CONVERSION_COM && parser.parsedResult.platform.type === 'desktop'
-        || e === PREVIEW_DIS && parser.parsedResult.platform.type === 'desktop') {
+    if (e === CONVERSION_COM && !isMobile
```

Comment:
for the chrome extension, have we considered using [user agent client hints](https://web.dev/migrate-to-ua-ch/)?  It's not supported on all browsers, but it should be supported on the browsers we need (Chrome, new Edge)